### PR TITLE
Add logging to diagnose an issue with hidden windows dialog

### PR DESCRIPTION
**Pull request by jakubl7545**
@lukaszgo1

### CODE REVIEW

**Review by lukaszgo1 — CHANGES_REQUESTED** [review]
This is a nice start! I'd suggest adding some more verbose logging into `splitToStacks`, to record what stacks are created, what windows end-up in a given stack and what finally gets yielded from this method. It is also helpful to be more descriptive in the logged info i.e. in addition to indexes log a textual description of what exactly is being logged. Note that while its fine to send it like this to the affected user, before this is merged to master we should make sure the info is logged at the debug level.

**Comment by jakubl7545** [comment]
@lukaszgo1 I've added more logging with descriptions. If it's good I'll send Wizard to the user who reported this bug and maybe something will come from it.

**Review by lukaszgo1 — COMMENTED** [review]
This is fine for sending to the user. When we will understand the cause I'll take another look before merging to master.

File: addon/globalPlugins/winWizard.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # Currently maitained by Jakub Lukowicz <jakubl7545@gmail.com>
 
 from __future__ import annotations
-
```
Comment (lukaszgo1) [inline addon/globalPlugins/winWizard.py:11]: Please re-introduce this empty line for readability.

File: addon/globalPlugins/winWizard.py [hunk 2]
```diff
@@ -403,6 +404,7 @@ def splitToStacks(self):
 			raise RuntimeError("Cannot divide empty list into stacks")
 		else:
 			slotsToStacks = [int(str(slotNumber + 10)[:-1]) for slotNumber in takenSlots]
+			log.info(f'Stacks with hidden windows: {sorted(set(slotsToStacks))}')
```
Comment (lukaszgo1) [inline addon/globalPlugins/winWizard.py:407]: Please assign this to a variable - something like `stacksWithHiddenWindows` and then use it in the logging call, and in the loop below.

File: addon/globalPlugins/winWizard.py [hunk 3]
```diff
@@ -415,6 +417,8 @@ def splitToStacks(self):
 							str(self[windowNumber])
 						)
 					)
+				log.info(f'Windows hidden in the same stack: {hiddenInCurrentStack}')
+				log.info(f'Yielded stack number: {Stack(stackNumber, sorted(hiddenInCurrentStack, key=lambda slot: slot.presentationalNumber))}')
 				yield Stack(stackNumber, sorted(hiddenInCurrentStack, key=lambda slot: slot.presentationalNumber))
```
Comment (lukaszgo1) [inline addon/globalPlugins/winWizard.py:422]: Same here i.e. assign into a variable, use it when logging and then finally yield.

**Review by lukaszgo1 — COMMENTED** [review]
Please convert all `log.info` calls to `log.debug`.

**Review by lukaszgo1 — APPROVED** [review]
LGTM! Feel free to squash merge.

**Pull request merged**